client/src/main.ts

```diff
@@ -96,7 +96,11 @@ export async function convertScript(src: string, opts: Partial<VuedcOptions> = {
   }
 
   if (opts.stopOnCollisions && hasCollisions()) {
-    throw new VuedcError(getCollisionsWarning(false));
+    const collisionsWarning = getCollisionsWarning(false);
+
+    resetRegistry();
+    
+    throw new VuedcError(collisionsWarning);
```

Comment:
These whitespace characters are frustrating, sorry. I am going to modify the code actions to just perform prettier format and write in the checks to avoid this sort of back and forth in the future.

```suggestion
    const collisionsWarning = getCollisionsWarning(false);

    resetRegistry();

    throw new VuedcError(collisionsWarning);
```